<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-5G75CIH" failures="1" tests="4" name="Test" time="14.47" errors="0" timestamp="2023-09-21T13:24:56 IST">
  <testcase classname="com.testUtility.ApplyingForAc" name="applyingForAc" time="0.505">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.testUtility.ApplyingForAc.applyingForAc(ApplyingForAc.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- applyingForAc -->
  <testcase classname="com.testUtility.ApplyingForAc" name="demo" time="0.002"/>
  <testcase classname="com.testUtility.ApplyingForAc" name="test" time="0.011"/>
  <testcase classname="com.testUtility.CreditCustomerTest" name="creditCustomer" time="4.07"/>
  <testcase classname="com.genericUtilities.BaseClass" name="@AfterSuite closeDb" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite closeDb -->
  <testcase classname="com.genericUtilities.BaseClass" name="@BeforeSuite connectDB" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite connectDB -->
</testsuite> <!-- Test -->
